aws/manageSSLCertificate.sh

```diff
@@ -21,13 +21,13 @@ Usage: $(basename $0) -i CERTIFICATE_ID -p CERTIFICATE_PUBLIC -v CERTIFICATE_PRI
 
 where
 
-(c) -c CERTIFICATE_CHAIN is the path to the file containing intermediate certificates, required for upload operation
-    -h shows this text
-(c) -i CERTIFICATE_ID is the id of the certificate, required for delete and upload operations
-(c) -p CERTIFICATE_PUBLIC is the path to the public certificate file, required for upload operation
-(o) -q minimal output (quiet)
-(c) -r REGION is the AWS region identifier for the region where the certificate should be updated, required for upload operation
-(c) -v CERTIFICATE_PRIVATE is the path to the private certificate file, required for upload operation
+(c) -c CERTIFICATE_CHAIN    is the path to the file containing intermediate certificates, required for upload operation
+    -h                      shows this text
+(c) -i CERTIFICATE_ID       is the id of the certificate, required for delete and upload operations
+(c) -p CERTIFICATE_PUBLIC   is the path to the public certificate file, required for upload operation
+(o) -q                      minimal output (quiet)
+(c) -r REGION               is the AWS region identifier for the region where the certificate should be updated, required for upload operation
+(c) -v CERTIFICATE_PRIVATE  is the path to the private certificate file, required for upload operation
 (o) -l (CERTIFICATE_OPERATION=${CERTIFICATE_OPERATION_LIST}) to list the certificates
 (o) -d (CERTIFICATE_OPERATION=${CERTIFICATE_OPERATION_DELETE}) to delete the certificate
 
```

Comment:
I ignored these two last records as indent would be too big for the previous ones